### PR TITLE
Monitor the load average of the machine and display it in the api

### PR DESCRIPTION
**Pull request by ph**
This PR add new information in the /_node/stats api and will return the
load average of the machine in the following formats depending of the
platforms that logstash is running on:

**Linux**
```json
{
    "cpu" : {
      "percent" : 26,
      "load_average" : {
        "1m" : 2.826171875,
        "5m": 1.8261718,
        "15m": 1.56566
      }
    }
}
```

**MacOS and other platform that the OperatingMXBean understand**
```json
{
    "cpu" : {
      "percent" : 26,
      "load_average" : {
        "1m" : 2.826171875,
      }
    }
}
```

Load average is not available on Windows

Fixes: https://github.com/elastic/logstash/issues/6215

### CODE REVIEW

File: logstash-core/spec/api/lib/api/node_stats_spec.rb [hunk 1]
```diff
@@ -69,7 +69,8 @@
       },
       "cpu"=>{
         "total_in_millis"=>Numeric,
-        "percent"=>Numeric
+        "percent"=>Numeric,
+        "load_average" => { "1m" => Numeric }
```
Comment (suyograo) [inline logstash-core/spec/api/lib/api/node_stats_spec.rb]: why only `1m`?

File: logstash-core/lib/logstash/instrument/periodic_poller/jvm.rb [hunk 2]
```diff
@@ -33,11 +33,57 @@ def self.get(gc_name)
       end
     end
 
+    class LoadAverage
```
Comment (suyograo) [inline logstash-core/lib/logstash/instrument/periodic_poller/jvm.rb]: Can we move this to a separate file? it's cleaner, IMO

Comment (ph) [inline logstash-core/lib/logstash/instrument/periodic_poller/jvm.rb]: ++

File: logstash-core/spec/api/lib/api/node_stats_spec.rb (hunk 1 above)
Comment (suyograo) [inline logstash-core/spec/api/lib/api/node_stats_spec.rb]: @ph not worth it. this is good

**Comment by ph** [comment]
@suyograo I've move the LoadAverage class into his own file, same with the specs.

**Comment by suyograo** [comment]
LGTM

**Pull request closed without merging**